demo/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     compileSdkVersion project.ext.compileSdkVersion
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 24
```

Comment:
Reason why I am using minSdkVersion 24 is because project is using lambdas, and lambdas are supported in java 8 which require Android Nugat (API level 24), there are two options, 1 to keep the min SDK version to 24 or seconds to revert to 16 and change code compliance to not use lambdas and use java 7